README.md

```diff
@@ -1,6 +1,77 @@
 # LLM Compressor
+Easy to use model compression library that supports a growing list of formats including
+int and float quantization, activation quantization, and 2:4 sparsity.
 
-A library for compressing large language models utilizing the latest techniques and research in the field for both training aware and post training techniques. 
-The library is designed to be flexible and easy to use on top of PyTorch and HuggingFace Transformers, allowing for quick experimentation.
+## Overview
+A library for compressing large language models utilizing the latest techniques and research for both training aware and post training techniques.
+llm-compressor is designed to be flexible and easy to use on top of PyTorch and HuggingFace Transformers, allowing for quick experimentation.
```

Comment:
I think here we should include

List of supported formats:
- Mixed Precision: W4A16 + W8A16
- Integer Quantization: W8A8 (int8)
- Floating Point Quantization: W8A8 (fp8)
- 2:4 Semi-structured Sparsity
- Unstructured Sparsity

List of Supported Techniques:
- PTQ
- GPTQ
- SmoothQuant
- SparseGPT
- QAT (coming soon)